### PR TITLE
Docs. Direct invite link via docs

### PR DESCRIPTION
**Pull request by benarent**
This PR changes the flow of trying labs. There is now a dedicated invite link for docs and skips a couple of clicks for users. Feedback from @stevenGravy .

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/try-out-teleport/browser-labs.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ You can quickly try out some of Teleport's key features from your browser.
 
 Choose one of our interactive learning tracks, which are hosted by Instruqt:
 
-- [Try Teleport Server Access](https://goteleport.com/labs/teleport-demo/): Try Teleport Server Access, which makes it easier to configure onboarding, RBAC, and auditing for SSH connections to remote hosts.
-- [Try Teleport Application Access](https://goteleport.com/labs/teleport-app-demo/): Try Teleport Application Access, which gives you secure access to your internal web applications.
-- [Try Teleport Database Access](https://goteleport.com/labs/teleport-database-access-demo/): Learn how to use Teleport to securely access and configure a remote MySQL database.
-- [Try Teleport Kubernetes Access](https://goteleport.com/labs/teleport-kubernetes-demo/): Try Teleport Kubernetes Access, which provides advanced RBAC controls and auditing for `kubectl` commands.
+- [Try Teleport Server Access](https://play.instruqt.com/teleport/invite/imz8g1n1xzru): Try Teleport Server Access, which makes it easier to configure onboarding, RBAC, and auditing for SSH connections to remote hosts.
```
Comment (ptgott) [inline docs/pages/try-out-teleport/browser-labs.mdx:11]: Now that all of these links point to the same URL, should we condense them into a single link instead? E.g., "Choose one of our interactive learning tracks, which are hosted by Instruqt:" becomes:

```markdown
Try our [interactive learning tracks](https://play.instruqt.com/teleport/invite/imz8g1n1xzru), which are hosted by Instruqt.
```

Comment (benarent) [reply docs/pages/try-out-teleport/browser-labs.mdx:11]: I think it's better to list all tracks, even with the same link so it's clear which tracks are available to try.

Comment (alexfornuto) [reply docs/pages/try-out-teleport/browser-labs.mdx:11]: You're both right. Having multiple copies of the same link can disorient the reader, but listing available labs is a good thing. How about something like this:

```
Choose one of our [interactive learning tracks](https://play.instruqt.com/teleport/invite/imz8g1n1xzru), which are hosted by Instruqt. These labs cover:

- Teleport Server Access, which makes it easier to configure onboarding, RBAC, and auditing for SSH connections to remote hosts.
- Teleport Application Access, which gives you secure access to your internal web applications.
- Teleport Database Access, to learn how to securely access and configure a remote MySQL database.
- Teleport Kubernetes Access, which provides advanced RBAC controls and auditing for `kubectl` commands.
```

Comment (benarent) [reply docs/pages/try-out-teleport/browser-labs.mdx]: Thanks, @alexfornuto and @ptgott . I've taken your feedback onboard and updated the PR. PTAL.

File: docs/pages/index.mdx [hunk 2]
```diff
@@ -23,13 +23,14 @@ Quickly see how Teleport works in one of our demo environments.
...
   Ocean droplet to set up Teleport quickly.
-- [Browser Labs](https://goteleport.com/labs/): Try Teleport using our guided
+- [Browser Labs](https://play.instruqt.com/teleport/invite/imz8g1n1xzru): Try Teleport using our guided
   interactive labs.
 - [Docker Compose Lab](./try-out-teleport/docker-compose.mdx): Try Teleport
   locally using Docker Compose.
 - [Kubernetes Lab](./try-out-teleport/local-kubernetes.mdx): See how Teleport
   runs on Kubernetes with this local lab.
 
+
```
Comment (alexfornuto) [inline docs/pages/index.mdx:33]: Removes extraneous newline.

```suggestion
```

**Review by alexfornuto — APPROVED** [review]
👍 with one minor nit.

**Pull request merged**